glommio/src/error.rs

```diff
@@ -473,6 +498,9 @@ impl<T> From<GlommioError<T>> for io::Error {
             GlommioError::IoError(io_err) => io_err,
             GlommioError::WouldBlock(_) => io::Error::new(io::ErrorKind::WouldBlock, display_err),
             GlommioError::Closed(_) => io::Error::new(io::ErrorKind::BrokenPipe, display_err),
+            GlommioError::CanNotBeClosed(_, _) => {
+                io::Error::new(io::ErrorKind::BrokenPipe, display_err)
```

Comment:
I don't know if this is the correct ErrorKind but I also don't have a better one in mind.. The thing about broken pipe is it seems to usually be an unfixable situation. While if we can't close this lock now, it might be close-able later. But maybe this is the best we can do.. I'll have to look at all the variants later.

Reply:
What do you think about using "ResourceBusy" for this? I think it makes some sense..

Reply:
Hi, @bryandmc ResourceBusy is part of an unstable build. Do not think it could be used. Maybe you have other options in your list?

Reply:
I can't believe I didn't see this! I feel silly haha ok after looking again, not in my editor on nightly, I think this might be our only decent choice actually. Ok I think we are stuck with this until better ones get stabilized.